packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -24,7 +24,6 @@ import * as url from 'url';
 import * as uuid from 'uuid';
 
 
-// TODO: maybe we should have a setup as a Client or as Server.
```

Comment:
Has this TODO been addressed? I don't see any change that reflects that it has.

Reply:
Yes, there is no need to have specific setups for client and servers.  No action to do.